server/controller/Songs/attribute.js

```diff
@@ -0,0 +1,7 @@
+export const SONG_ATTRIBUTE = {
+    songID: `songID`,
```

Comment:
please delete that when we don't need that

Reply:
I'll leave it there for future use

Reply:
In our database already generate automatically the ID, so we can delete that. Thanks @Dev01-vinamr-Ken

Reply:
No

Reply:
ok no problem with that